app/controllers/application_controller.rb

```diff
@@ -11,6 +11,7 @@ class ApplicationController < ActionController::Base
 
   rescue_from ActionController::InvalidAuthenticityToken, with: :invalid_auth_token
   rescue_from ActionController::UnknownFormat, with: :render_not_found
+  rescue_from Twilio::REST::RestError, with: :invalid_phone_number
```

Comment:
we don't want this twlio thing at the high level applicaiton controller -- we want to rescue in each of the controllers

I think this belongs in `TwoFactorAuthenticatable` in the `included` block